lib/actuators/include/Servo.h

```diff
@@ -1,9 +1,16 @@
 #pragma once
 
-#include "mbed.h"
+#include "Actuator.h"
 
-class Servo {
+class Servo : public Actuator {
```

Comment:
Servo should be a purely virtual interface. this means no constructors and no actual function definitions. The cherno's video on C++ Interfaces is basically what u should do

Reply:
Should helper functions like getSign() and functions that are common to all servos (like setPeriod() ) be made purely virtual in the base Servo class and be defined separately in the subclasses?

Reply:
Also I wouldn't be able to instantiate the m_pwm and m_pin members of Servo without the Servo constructor. Should those member variables be moved under the derived classes instead?

Reply:
getsign shouldnt be a function. just do (var >= 0 ? 1 : -1)

Reply:
getsign shouldnt be a function. its a one liner and it makes sense what youre doing. just do var >=0 ? 1 : -1;